installer/ansible/group_vars/common.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+# Dummy variable to avoid error because ansible does not recognize the
+# file as a good configuration file when no variable in it.
+dummy:
+
+
+###########
+# GENERAL #
+###########
+
+# This field indicates local machine host ip
+# For HA configuration, add the IP on which the Core Services (Hotpot) are reachable from the Kubernetes node.
+host_ip: 127.0.0.1
+
+#Deploy multicloud in HA mode - default is false 
+gelato_ha: false
+
+# This field indicates which way user prefers to install, currently support
+# 'repository', 'release' and 'container'
+install_from: release
+
+# Gelato- HA / Multi-Cloud HA installation is supported only from release / container. 
+install_gelato_ha_from: release   
+
+# These fields below will specify the tag based on install_from type
+repo_branch: master
+release_version: v1.7.0
+
+# This field indicates which os family the system will be running, currently
+# support 'Debian' and 'RedHat'
+ansible_os_family: Debian
+
+# Install as systemd process or standalone application: true / false
+# true: Will install api, dock, controller and etcd processes as a systemd services and enable them. Recommended for production use
+# false: Will install api, dock, controller and etcd as standalone processes. Recommended for development use.
+install_as_systemd: true
+
+# delete all source packages
+source_purge: true
+
+# delete database
+database_purge: true
+
+# OpenSDS configuration file
```

Comment:
this line may not required. plz check

Reply:
It is being used to remove etcd data directory while cleaning.
https://github.com/sodafoundation/multi-cloud/blob/b6459548812ed643e9a0e8f1186f5d341e86cab2/installer/ansible/roles/cleaner/handlers/main.yml#L39-L47